### PR TITLE
[IMP] web_advanced_search_x2x spanish translation

### PR DESCRIPTION
**Pull request by antespi**
Translation to spanish for addon web_advanced_search_x2x
I couldn't find this project in Transifex

@pedrobaeza thanks for your review

### CODE REVIEW

File: web_advanced_search_x2x/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#, python-format
+msgid "Use criteria"
+msgstr "Usar criterio"
+
+#. module: web_advanced_search_x2x
+#. openerp-web
+#: code:addons/web_advanced_search_x2x/static/src/js/web_advanced_search_x2x.js:159
+#, python-format
+msgid "invalid search domain"
+msgstr "filtro de búsqueda inválido"
```
Comment (pedrobaeza) [inline web_advanced_search_x2x/i18n/es.po]: s/filtro/dominio

**Comment by antespi** [comment]
@pedrobaeza done, thanks

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
:+1:

**Comment by antespi** [comment]
Please @pablocm-aserti, @Yajo, could you check this translation?
Thanks so much

**Comment by JavierIniesta** [comment]
:+1:

**Pull request merged**